.github/workflows/release-proposal.yml

```
@@ -0,0 +1,20 @@
+name: Release Proposal PR check
+
+on:
+  pull_request:
+    branches:
+      - v2.x
+      - v3.x
```

@Qard your are right about the number of files,updated, here and in sample PRs.
We will not have to create new file for each version, but we should add new branch in this list for each version

```suggestion
on:
  pull_request:
    branches:
      - v2.x
      - v3.x
      - v4.x
      - v5.x
      - ...
```

---

Why not just use a pattern?

---

Because I didn't know that it was possible 🤦  changed